docs/cardinal/commands.mdx

```diff
@@ -0,0 +1,179 @@
+
+You can get a command's persona using `Persona`:
+
+```go
+func AttackSystem(state *AttackSystemState) error {
+    for cmd := range state.AttackCommands.Iter() {
+        attack := cmd.Payload()
+        sender := cmd.Persona()
+
+        // Get the player's entity and verify ownership.
+        player, ok := state.Players.GetByID(attack.PlayerID)
+        if !ok {
+            continue
+        }
+        if player.Owner.Get() != sender {
+            continue // Skip unauthorized commands.
+        }
+
+        // Process the command...
+    }
+    return nil
+}
+```
+
+## Sending Commands
+
+Use the client SDK to send commands to the server:
+
+<Tabs>
+  <Tab title="C#">
+    ```csharp
+    await shard.SendCommandAsync("attack", new Dictionary<string, object>
+    {
+        ["target_id"] = "enemy-123",
+        ["damage"] = 50
+    });
+    ```
+  </Tab>
+  <Tab title="TypeScript">
+    ```ts
+    TODO: update
+    ```
+  </Tab>
+</Tabs>
+
+### Command-Reply Pattern
+
+Commands are asynchronous by design—when you send a command, you receive an acknowledgment, but not the result of the command's execution. If you need a result, you typically emit an event from the system that processes the command and subscribe to it from the client.
+
+Some command types are more synchronous in nature, like updating game config, where every command has a corresponding result event. For these cases, use the synchronous command method, which handles the temporary event subscription for you and returns the result event directly:
+
+<Tabs>
+  <Tab title="C#">
+    ```csharp
+    var result = await shard.SendCommandWithReplyAsync<PurchaseResult>("buy-item", new Dictionary<string, object>
+    {
+        ["item_id"] = "sword-01",
+        ["quantity"] = 1
+    });
+    ```
+  </Tab>
+  <Tab title="TypeScript">
+    ```ts
+    TODO: update
+    ```
+  </Tab>
+</Tabs>
+
+## Inter-Shard Commands
+
+In a multi-shard setup, you can send commands from within a system to another shard. This allows you to trigger systems in other shards or coordinate game state.
+
+### Defining Shard Targets
+
+To send an inter-shard command, you must first define an `OtherWorld` variable for each shard you want to communicate with:
+
+```go
+import "github.com/argus-labs/world-engine/pkg/cardinal"
+
+var MatchmakingShard = cardinal.OtherWorld{
+    Region:       "us-west-2",
+    Organization: "my-org",
+    Project:      "my-game",
+    ShardID:      "matchmaking",
+}
+```
+
+An `OtherWorld` represents a specific remote shard and is used to route commands to the correct destination.
+
+### Sending Inter-Shard Commands
+
+Use `SendCommand` to dispatch a command to another shard. Pass the system's `BaseSystemState` and the command you want to send:
+
+```go
+import matchcmd "my-game/shards/matchmaking/command"
```

Comment:
cmiiw, but shouldn’t this be in the `pkg` folder?

Reply:
it doesn't have to be.